a/apache-flink/apache-flink_master_rhel_8.3.sh

```diff
@@ -18,23 +18,68 @@ VERSION="master"
 
 # Install dependencies and tools.
 sudo yum update -y
-sudo yum install -y git wget java-1.8.0-openjdk-devel
+sudo yum install -y git wget java-1.8.0-openjdk-devel patch xz
 export JAVA_HOME=export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
 
-#Install maven v3.2.5 which is recommended by Flink 1.8.x
-wget https://www-us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz --no-check-certificate --quiet
-tar xzf apache-maven-3.2.5-bin.tar.gz
+# Install postgresql build dependencies
+sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-ppc64le/pgdg-redhat-repo-latest.noarch.rpm
+sudo yum install -y postgresql10-libs
+sudo yum install -y postgresql10 postgresql10-server postgresql10-contrib
 
-export M2_HOME=`pwd`/apache-maven-3.2.5
-export PATH=`pwd`/apache-maven-3.2.5/bin:${PATH}
+# Install maven v3.6.3
+wget https://www-us.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz --no-check-certificate --quiet
+tar xzf apache-maven-3.6.3-bin.tar.gz
 
-#Clone and build source
+# Set ENV variables
+export M2_HOME=`pwd`/apache-maven-3.6.3
+export PATH=`pwd`/apache-maven-3.6.3/bin:${PATH}
+
+# Clone and build source
 git clone https://github.com/apache/flink.git
 cd flink
 git checkout $VERSION
 
 # Updated com.google.protobuf:protoc from 3.5.1-->3.7.0
 sed -i 's/3.5.1/3.7.0/g' flink-formats/flink-parquet/pom.xml
 
-#Compile and build package using threads
+# Compile and build package using threads
 mvn clean package -T 6 -DskipTests -Dfast
+
+# Resolve "No postgres binary for ppc64le" error
+mkdir tmpdir
+cd tmpdir
+rm -rf *
+mv ~/.m2/repository/com/opentable/components/otj-pg-embedded/0.13.3/otj-pg-embedded-0.13.3.jar .
+jar xf otj-pg-embedded-0.13.3.jar
+rm -rf otj-pg-embedded-0.13.3.jar
+mkdir postgresql-Linux-ppc64le
+tar xf postgresql-Linux-x86_64.txz -C postgresql-Linux-ppc64le
+cd postgresql-Linux-ppc64le
+rm -rf bin/*
+rm -rf lib/*
+rm -rf share/*
+cp /usr/pgsql-10/bin/pg_ctl bin/
+cp /usr/pgsql-10/bin/postgres bin/
+cp /usr/pgsql-10/bin/initdb bin/
+cp -r /usr/pgsql-10/lib/* lib/
+cp -r /usr/pgsql-10/share/* share/
+tar cJf postgresql-Linux-ppc64le.txz share lib bin
+mv postgresql-Linux-ppc64le.txz ../
+cd ..
+jar cf otj-pg-embedded-0.13.3.jar com META-INF otj-pg-embedded.properties postgresql-Linux-ppc64le.txz postgresql-Darwin-x86_64.txz postgresql-Linux-x86_64.txz postgresql-Windows-x86_64.txz
+cp otj-pg-embedded-0.13.3.jar ~/.m2/repository/com/opentable/components/otj-pg-embedded/0.13.3/
+cd ..
+
+# Create /var/run/postgresql and change ownership to current non-root user
+USER=`whoami`
+mkdir -p /var/run/postgresql/
+sudo chown $USER:$USER /var/run/postgresql/
+
+# Updated pom.xml with known failures (Tests which is same as x86_64)
+wget https://github.com/ManirajDeivendran/build-scripts/blob/master/a/apache-flink/apache-flink_ignore_test_fail_as_x86.patch
```

Comment:
I understand the failures are common to x86 and ppc64le, so lets not mention "x86" in the name, it can be confusing